docs/swagger.json

```diff
@@ -870,7 +870,8 @@
                 "organization",
                 "skills",
                 "slack_username",
-                "username"
+                "username",
+                "is_available"
```

Comment:
She'll we modify swagger.json manually?

Reply:
I had done this in a previous task. The PR template also has a field ```Update Swagger documentation and the exported file at /docs folder```. Is there any way to automatically generate it?